it/uk/gov/hmrc/minorentityidentification/connectors/RegisterWithMultipleIdentifiersConnectorISpec.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.minorentityidentification.connectors
+
+import play.api.test.Helpers._
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.minorentityidentification.assets.TestConstants.{testRegime, testSafeId, testSautr}
+import uk.gov.hmrc.minorentityidentification.connectors.RegisterWithMultipleIdentifiersHttpParser.RegisterWithMultipleIdentifiersSuccess
+import uk.gov.hmrc.minorentityidentification.featureswitch.core.config.{DesStub, FeatureSwitching}
+import uk.gov.hmrc.minorentityidentification.stubs.{AuthStub, RegisterWithMultipleIdentifiersStub}
+import uk.gov.hmrc.minorentityidentification.utils.ComponentSpecHelper
+
+class RegisterWithMultipleIdentifiersConnectorISpec extends ComponentSpecHelper with AuthStub with RegisterWithMultipleIdentifiersStub with FeatureSwitching {
+
+  lazy val connector: RegisterWithMultipleIdentifiersConnector = app.injector.instanceOf[RegisterWithMultipleIdentifiersConnector]
+
+  private implicit val headerCarrier: HeaderCarrier = HeaderCarrier()
+
+  "registerWithMultipleIdentifiers" when {
+    s"the $DesStub feature switch is disabled" when {
```

Comment:
I think this
s"the $DesStub feature switch is disabled" when { 
should be 
s"the $DesStub feature switch is disabled" should { 
Also maybe add a test for when the call returns a different status (e.g. 500 or 400) and check the InternalServerException is being thrown

Reply:
updated and added